ouroboros-consensus/src/Ouroboros/Consensus/BlockchainTime/WallClock/Util.hs

```diff
@@ -21,6 +21,7 @@ import           Control.Monad.Class.MonadTimer (MonadDelay (..))
 
```

Comment:
Remove this import (and import `MonadDelay` from `IOLIke`, because `threadDelay` is now still in scope).